src/verify/PhoneVerificationStartScreen.tsx

```diff
@@ -0,0 +1,293 @@
+import { Countries } from '@celo/utils/lib/countries'
```

Comment:
this file is mostly copy paste + remove some complexity from the VerificationEducationScreen component

Reply:
Naively I was thinking we could slightly branch in the existing `VerificationEducationScreen`.
But looks like this approach is better. There's quite some complexity there already.